eq-author/src/App/QuestionnairesPage/QuestionnairesView/reducer.js

```diff
@@ -55,6 +70,18 @@ const reducer = (state, action) => {
         ...state,
         autoFocusId: calculateAutoFocusId(state.questionnaires, action.payload),
       };
+    case ACTIONS.SORT_COLUMN:
+      buildInitialState(state.questionnaires)(state);
```

Comment:
I was expecting this to call `sortQuestionnaires`? So chunking would be moved there as well. Also, its changes to questionnaires and pages needs to be returned to update the state.